tools/ci.psm1

```diff
@@ -593,3 +584,176 @@ function Invoke-AppveyorFinish
         throw $_
     }
 }
+
+function Invoke-LinuxTests
+{
+    $createPackages = $false
+    $isFullBuild = $hasFeatureTag
+    Write-Host -Foreground Green "Executing ci.psm1 -BootStrap `$isPR='$isPr' - $commitMessage"
+    # Make sure we have all the tags
+    Sync-PSTags -AddRemoteIfMissing
+    Start-PSBootstrap -Package:$createPackages
```

Comment:
Split bootstrap into a separate function and call it in a separate task from the YAML.

Reply:
Okay

Reply:
What line does the bootstrap phase end on?